### PR TITLE
taxonomy: new categories and Lithuanian translations

### PR DESCRIPTION
**Pull request by DanieliusAsm**
### What

- Added new categories in Beers
- Added new categories in Non-alcoholic beers
- Added non-alcoholic ciders

Product examples for some of the categories:
 - [Non alcoholic pilsner](https://lt.openfoodfacts.org/product/4770349231323/nealkoholinis-alus-pilsner-%C5%A1vyturys)
 - [Non alcoholic lager](https://world-lt.openfoodfacts.org/product/4770349232825/nealkoholinis-alus-tuborg)
 - [Non alcoholic radler](https://lt.openfoodfacts.org/product/4770301237332/nealkoholinis-alaus-kokteilis-su-citrin%C5%B3-sultimis-volfas-engelman)
 - [Non alcoholic APA](https://lt.openfoodfacts.org/product/4779043540380/nealkoholinis-american-pale-ale-genys)
 - [Non alcoholic Organic beer ](https://lt.openfoodfacts.org/product/4770349233082/ekologi%C5%A1kas-nealkoholinis-alus-carlsberg)
 - [Non alcoholic Cherry Kriek](https://lt.openfoodfacts.org/product/4770301233297/nealkoholinis-vy%C5%A1ni%C5%B3-kriek-volfas-engelman)
 - [non Alcoholic ciders](https://world.openfoodfacts.org/product/5900014005044/wild-berries-somersby)
 - [American Pale ale](https://lt.openfoodfacts.org/product/4779043540342/american-pale-ale-genys)
 - [Danish beers](https://world-lt.openfoodfacts.org/brand/carlsberg)
 - [Latvian beers](https://lt.openfoodfacts.org/product/4750132006956/nefiltruotas-l%C4%81%C4%8Dpl%C4%93sis)
 - [Polish beers](https://lt.openfoodfacts.org/brand/zywiec)

Im not so certain about APA categories since there will be very few products. Maybe these prodcuts should just go under [Special beers](https://world.openfoodfacts.org/category/special-beers) category instead.

### CODE REVIEW

**Comment by DanieliusAsm** [comment]
I have seen some beers as APA (America Pale Ale) and AIPA (American India Pale Lager) The latter should be a category under IPA, but not sure if those 2 are the same or different beers.

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -1562,11 +1809,13 @@ origins:en: fr:Bretagne
 wikidata:en:Q2905049
 
 <en:Beers
-en:Beers from Ireland, Irish beers
+<en:Country specific beers
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:1812]: Should we keep Irish beers directly under Beers, or have it under Country specific beer, which is itself under Beers? If we want the latter, then we need to remove the <en:Beers from Irisih beers etc.

Comment (DanieliusAsm) [reply taxonomies/categories.txt:1812]: Took me a while to understand. But removed it in [699ec3e](https://github.com/openfoodfacts/openfoodfacts-server/pull/8718/commits/699ec3e7539ad8f6c7d7c17b690005df18620b82)

**Comment by DanieliusAsm** [comment]
Mistake in commit name it should have said: "porter,stouts under dark beers"

**Review by DanieliusAsm — COMMENTED** [review]
There are 2 vegetable cakes categories in [salty snacks](https://world.openfoodfacts.org/category/salty%20snacks)

**Comment by alexgarel** [comment]
@DanieliusAsm there is a problem to fix:

```
building taxonomy for categories - publish: 1
ERROR - lt:belgiškas-alus already is associated to en:dutch-beers (categories) - lt:belgiškas-alus cannot be mapped to entry en:belgian-beers
[Fri Aug  4 17:41:00 2023] build_lang.pl: Errors in the categories taxonomy definition at /opt/product-opener/lib/ProductOpener/Tags.pm line 1386.
[Fri Aug  4 17:41:00 2023] build_lang.pl: Compilation failed in require at /opt/product-opener/scripts/build_lang.pl line 31.
<h1>Software error:</h1>
[Fri Aug  4 17:41:00 2023] build_lang.pl: BEGIN failed--compilation aborted at /opt/product-opener/scripts/build_lang.pl line 31.
<pre>Errors in the categories taxonomy definition at /opt/product-opener/lib/ProductOpener/Tags.pm line 1386.
Compilation failed in require at /opt/product-opener/scripts/build_lang.pl line 31.
BEGIN failed--compilation aborted at /opt/product-opener/scripts/build_lang.pl line 31.
</pre>
<p>
For help, please send mail to this site's webmaster, giving this error message 
and the time and date of the error.

</p>
255
```

**Pull request merged**